src/main.lua

```diff
@@ -1,112 +1,93 @@
 local STI = require "libs.STI"
-local avatar
 local playerX = 200
 local playerY = 200
-local speed = 200
+local playerspeed = 200
```

Comment:
Ok thanks I'll merge it later today after changing the speed and alignment stuff. I'm glad you think it's good!!